cw/capture.py

```diff
@@ -156,9 +158,12 @@ def init(ctx: typer.Context):
     initialize_capture(ctx.obj.cfg["device"], ctx.obj.cfg["capture"])
 
 
-def capture_init(ctx, num_traces, plot_traces):
+def capture_init(ctx, force_program_bitstream, num_traces, plot_traces):
     """Initializes the user data stored in the context and programs the target."""
     cfg = ctx.obj.cfg
+    if force_program_bitstream is not None:
```

Comment:
It is perhaps better to change this to `if force_program_bitstream:`

Reply:
But if I do that, it won't be possible to overwrite this option in CLI with `--no-force-program-bitstream` when it is already defined as True in yaml file.

Reply:
You're right. it wouldn't be possible to override this setting in command line.
OK, then i think this is ready to merge